docs/node-mixin/dashboards/network.libsonnet

```diff
@@ -0,0 +1,747 @@
+        legendFormat='{{device}} received',
+      ))
+      .addTarget(commonPromTarget(
+        'irate(node_network_transmit_compressed_total{%(nodeQuerySelector)s,}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='{{device}} transmitted',
+      ))
+      .withDecimals(1)
+      .withUnits('pps')
+      .withNegativeYByRegex('transmit')
+      .withAxisLabel('out(-) / in(+)'),
+
+    local networkMulticastPanel =
+      nodeTimeseries.new('Multicast Packets')
+      .addTarget(commonPromTarget(
+        'irate(node_network_receive_multicast_total{%(nodeQuerySelector)s,}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='{{device}} received',
+      ))
+      .addTarget(commonPromTarget(
+        'irate(node_network_transmit_multicast_total{%(nodeQuerySelector)s,}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='{{device}} transmitted',
+      ))
+      .withDecimals(1)
+      .withUnits('pps')
+      .withNegativeYByRegex('transmit'),
+
+    local networkFifoPanel =
+      nodeTimeseries.new('Network FIFO')
+      .addTarget(commonPromTarget(
+        'irate(node_network_receive_fifo_total{%(nodeQuerySelector)s,}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='{{device}} received',
+      ))
+      .addTarget(commonPromTarget(
+        'irate(node_network_transmit_fifo_total{%(nodeQuerySelector)s,}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='{{device}} transmitted',
+      ))
+      .withDecimals(1)
+      .withUnits('pps')
+      .withNegativeYByRegex('transmit')
+      .withAxisLabel('out(-) / in(+)'),
+
+    local networkNFConntrack =
+      nodeTimeseries.new('NF Conntrack')
+      .addTarget(commonPromTarget(
+        'node_nf_conntrack_entries{%(nodeQuerySelector)s}' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='NF conntrack entries',
+      ))
+      .addTarget(commonPromTarget(
+        'node_nf_conntrack_entries_limit{%(nodeQuerySelector)s}' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='NF conntrack limits',
+      ))
+      .withFillOpacity(0),
+
+    local networkSoftnetPanel =
+      nodeTimeseries.new('Softnet Packets')
+      .addTarget(commonPromTarget(
+        'irate(node_softnet_processed_total{%(nodeQuerySelector)s}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='CPU {{cpu }} proccessed',
+      ))
+      .addTarget(commonPromTarget(
+        'irate(node_softnet_dropped_total{%(nodeQuerySelector)s}[$__rate_interval])' % config { nodeQuerySelector: c.nodeQuerySelector },
+        legendFormat='CPU {{cpu }} dropped',
+      ))
+      .withDecimals(1)
+      .withUnits('pps')
+      .withNegativeYByRegex('dropped')
+      .withAxisLabel('Dropped(-) | Processed(+)'),
```

Comment:
Per previous comment RE: `out(-) / in(+)`. Pipe is used here. Whichever format is preferred should be used consistently.

Reply:
thanks, changed to `|` everywhere.